packaging/windows/build_pypi_pkg/setup.py

```diff
@@ -0,0 +1,64 @@
+import sys
+import subprocess
+
+import setuptools.command.install
+from setuptools import find_packages, setup
+import wheel.bdist_wheel
+
+pkg_name="torch"
+pkg_ver="{{GENERATE_TORCH_PKG_VER}}"
```

Comment:
Do we need to generate the source package for every release of PyTorch?

Reply:
This one is just for PT1.7. It is a temp solution to work round PyPI package size limitation.